addons/mail/static/src/owl/components/message/message.xml

```diff
@@ -57,9 +57,21 @@
             <div class="o_Message_core">
                 <t t-if="!props.isSquashed">
                     <div class="o_Message_header">
-                        <div class="o_Message_authorName" t-att-class="hasAuthorRedirect ? 'o_Message_authorRedirect o_redirect' : ''" t-on-click="_onClickAuthor" t-att-title="hasAuthorRedirect ? 'Open chat' : ''">
-                            <t t-esc="displayedAuthorName"/>
-                        </div>
+                        <t t-if="storeProps.author">
+                            <div class="o_Message_authorName" t-att-class="hasAuthorRedirect ? 'o_Message_authorRedirect o_redirect' : ''" t-on-click="_onClickAuthor" t-att-title="hasAuthorRedirect ? 'Open chat' : ''">
+                                <t t-esc="displayedAuthorName"/>
```

Comment:
replace component getter, since there's no longer a reason to keep it:
`storeGetters.partnerName(storeProps.author.localId)`